lib/DockerWrapper.js

```diff
@@ -61,6 +61,24 @@ const getCurrentServiceConfig = (service) => {
   });
 };
 
+const generateServiceEnvVars = async (app) => {
```

Comment:
Should this be called generateDatabaseEnvVars? That's all it does at the moment, right?

Reply:
This actually generates a full environment variable list: database env variables **and** custom environment variables. (See lines 75-78 for where we pull out custom environment variables with `...app.envVariables`)

Reply:
👍